### PR TITLE
created getUserInfo firebase function

### PR DESCRIPTION
**Pull request by wry0313**

### CODE REVIEW

File: web/functions/src/getUserInfo.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as functions from "firebase-functions";
+import * as admin from "firebase-admin";
```
Comment (egelja) [inline web/functions/src/getUserInfo.ts:2]:
```suggestion
import functions from "firebase-functions";
import admin from "firebase-admin";
```

does this not work?

File: web/functions/src/getUserInfo.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "An error occurred while retrieving user information.",
+      error.message
+    );
+  }
+});
+
+function apiKeyIsVerified(apiKey: string) {
+  // replace with actual api key value
+  return apiKey === 'your_expected_api_key';
+}
```
Comment (egelja) [inline web/functions/src/getUserInfo.ts:40]: huh

**Review by egelja — CHANGES_REQUESTED** [review]
is this PR ready for review?

File: web/functions/src/getUserInfo.ts (hunk 2 above)
Comment (wry0313) [reply web/functions/src/getUserInfo.ts:40]: So Steve will handle this part. I'm not sure what his approach would exactly be

**Pull request closed without merging**